Modules/_uuidmodule.c

```diff
@@ -91,45 +121,1730 @@ py_windows_has_stable_node(void)
+{
+    uuidobject *self = (uuidobject *)o;
+    return PyLong_FromUnsignedLong(get_time_mid(self));
+}
+
+static PyObject *
+Uuid_get_time_hi_version(PyObject *o, void *closure)
+{
+    uuidobject *self = (uuidobject *)o;
+    return PyLong_FromUnsignedLong(get_time_hi_version(self));
+}
+
+static PyObject *
+Uuid_get_clock_seq_hi_variant(PyObject *o, void *closure)
+{
+    uuidobject *self = (uuidobject *)o;
+    return PyLong_FromUnsignedLong(get_clock_seq_hi_variant(self));
+}
+
+static PyObject *
+Uuid_get_clock_seq_low(PyObject *o, void *closure)
+{
+    uuidobject *self = (uuidobject *)o;
+    return PyLong_FromUnsignedLong(get_clock_seq_low(self));
+}
+
+static PyObject *
+Uuid_get_time(PyObject *o, void *closure)
+{
+    uuidobject *self = (uuidobject *)o;
+
+    long version = get_version(self);
+
+    if (version == 6) {
+        // UUID v6: time_hi (32) | time_mid (16) | ver (4) | time_lo (12) | ... (64)
+        uint32_t time_hi = get_time_low(self);
+        uint16_t time_mid = get_time_mid(self);
+        uint16_t time_lo = ((uint16_t)(self->bytes[6] & 0x0f) << 8) |
+                          ((uint16_t)self->bytes[7]);
+
+        uint64_t time = ((uint64_t)time_hi << 28) |
+                        ((uint64_t)time_mid << 12) |
+                        (uint64_t)time_lo;
+        return PyLong_FromUnsignedLongLong(time);
+    }
+    else if (version == 7) {
+        // UUID v7: unix_ts_ms (48) | ... (80)
+        // First 6 bytes are the 48-bit timestamp
+        uint64_t unix_ts_ms = ((uint64_t)self->bytes[0] << 40) |
+                             ((uint64_t)self->bytes[1] << 32) |
+                             ((uint64_t)self->bytes[2] << 24) |
+                             ((uint64_t)self->bytes[3] << 16) |
+                             ((uint64_t)self->bytes[4] << 8) |
+                             ((uint64_t)self->bytes[5]);
+        return PyLong_FromUnsignedLongLong(unix_ts_ms);
+    }
+    else {
+        // UUID v1 and others: time_lo (32) | time_mid (16) | ver (4)
+        //                     | time_hi (12) | ... (64)
+        uint32_t time_lo = get_time_low(self);
+        uint16_t time_mid = get_time_mid(self);
+        uint16_t time_hi = ((uint16_t)(self->bytes[6] & 0x0f) << 8) |
+                          ((uint16_t)self->bytes[7]);
+
+        uint64_t time = ((uint64_t)time_hi << 48) |
+                        ((uint64_t)time_mid << 32) |
+                        (uint64_t)time_lo;
+        return PyLong_FromUnsignedLongLong(time);
+    }
+}
+
+static PyObject *
+Uuid_get_clock_seq(PyObject *o, void *closure)
+{
+    uuidobject *self = (uuidobject *)o;
+    // clock_seq_hi_variant (byte 8) & 0x3f, then clock_seq_low (byte 9)
+    uint16_t clock_seq = ((uint16_t)(get_clock_seq_hi_variant(self) & 0x3f) << 8) |
+                         ((uint16_t)get_clock_seq_low(self));
+    return PyLong_FromUnsignedLong(clock_seq);
+}
+
+static PyObject *
+Uuid_get_node(PyObject *o, void *closure)
```

Comment:
Another way to ease your life in terms of API calls is to use AC here so that you only need to write `return get_node(self);` (no cast is needed as AC will be smart enough) and you specify the return type to be `unsigned_long_long`. It will also allow us to write docstrings.